library/Icingadb/Model/State.php

```diff
@@ -78,4 +80,47 @@ public function createBehaviors(Behaviors $behaviors)
             'last_comment_id'
         ]));
     }
+
+    /**
+     * Get the state icon
+     *
+     * @return Icon|null
+     */
+    public function getIcon(): ?Icon
+    {
+        $icon = null;
+        if ($this->is_handled) {
+            switch (true) {
+                case $this->is_acknowledged:
+                    $icon = new Icon(Icons::IS_ACKNOWLEDGED);
+                    break;
+                case $this->in_downtime:
+                    $icon = new Icon(
+                        Icons::IN_DOWNTIME,
+                        ['title' => sprintf(
+                            '%s (%s)',
+                            strtoupper($this->getStateTextTranslated()),
+                            t('handled by Downtime')
+                        )]
+                    );
+                    break;
+                case $this->is_flapping:
+                    $icon = new Icon(Icons::IS_FLAPPING);
+                    break;
+                default:
+                    $icon = new Icon(Icons::HOST_DOWN);
+            }
+        } elseif ($this->in_downtime) {
```

Comment:
Please add this with `! $this->is_handled && $this->in_downtime` to the switch case above.